connector/src/main/java/org/jboss/as/connector/subsystems/datasources/DataSourceDefinition.java

```diff
@@ -196,6 +196,23 @@ protected boolean isValueDiscardable(PathAddress address, String attributeName,
                 .end();
     }
 
+    static void registerTransformers130(ResourceTransformationDescriptionBuilder parentBuilder) {
+        ResourceTransformationDescriptionBuilder builder = parentBuilder.addChildResource(PATH_DATASOURCE);
+        builder.getAttributeBuilder()
+                .setDiscard(new DiscardAttributeChecker.DefaultDiscardAttributeChecker() {
+                    @Override
+                    protected boolean isValueDiscardable(PathAddress address, String attributeName, ModelNode attributeValue, TransformationContext context) {
+                        return attributeValue.equals(new ModelNode(false));
+                    }
+                }, TRACKING)
+                .addRejectCheck(RejectAttributeChecker.DEFINED, TRACKING).end()
+                //We're rejecting operations when statistics-enabled=false, so let it through in the enable/disable ops which do not use that attribute
+                .addOperationTransformationOverride(DATASOURCE_ENABLE.getName())
+                .end()
+                .addOperationTransformationOverride(DATASOURCE_DISABLE.getName())
+                .end();
```

Comment:
@bstansberry I don't remember the details, but looking at the commit history in the comments in https://issues.jboss.org/browse/WFLY-3084 it looks like I was involved.

@maeste However, as Brian says 1.3.0 (6.3.0 and 6.4.0) supports statistics-enabled so perhaps whatever we are doing is not needed?